crates/sui-protocol-config/src/lib.rs

```diff
@@ -1453,6 +1461,10 @@ impl ProtocolConfig {
                         cfg.max_jwk_votes_per_validator_per_epoch = Some(240);
                         cfg.max_age_of_jwk_in_epochs = Some(1);
                     }
+
+                    if chain != Chain::Mainnet && chain != Chain::Testnet {
```

Comment:
Is how we do these sort of gatings normally? 
I'd feel more comfortable with 
```suggestion
                    match chain {
                        Chain::Mainnet | Chain::Testnet => (),
                        Chain::Unknown => cfg.feature_flags.shared_object_deletion = true,
                     }
```

Reply:
Or at least alternatively `if chain == Chain::Unknown` would make be feel better :) 
Just something more explicit